src/main/java/com/epam/reportportal/auth/integration/github/GithubOAuthProvider.java

```diff
@@ -44,28 +45,38 @@
 
     private final GitHubUserReplicator githubReplicator;
     private final AuthConfigService authConfigService;
+    private final String tokenUrl;
+    private final String authUrl;
+    private final String githubBaseUrl;
 
     public GithubOAuthProvider(GitHubUserReplicator githubReplicator,
-                               AuthConfigService authConfigService) {
+                               AuthConfigService authConfigService,
+                               @Value("${rp.auth.github.tokenUrl:https://github.com/login/oauth/access_token}") String tokenUrl,
```

Comment:
Please, put the default value into the .yaml file.

Reply:
updated on develop branch